Projects/B-L475E-IOT01A/Applications/WiFi/Common/Src/es_wifi_io.c

```diff
@@ -210,6 +210,11 @@ int8_t SPI_WIFI_ResetModule(void)
 
   while (WIFI_IS_CMDDATA_READY())
   {
+    if (count > 4)
```

Comment:
Should `count` not be compared to 6 instead of 4? See [this](https://github.com/STMicroelectronics/STM32CubeL4/blob/503ab076587f3241e5d8f25adea452dbf80a8b4f/Projects/B-L475E-IOT01A/Applications/WiFi/Common/Src/es_wifi_io.c#L203) declaration of array `Prompt[]`.